core/SConscript.bootloader

```diff
@@ -130,6 +138,29 @@ if TREZOR_MODEL in ('R'):
 if TREZOR_MODEL in ('T',):
     SOURCE_TREZORHAL.append('embed/trezorhal/touch.c')
 
+if DMA2D:
+    SOURCE_STMHAL.append('vendor/micropython/lib/stm32lib/STM32F4xx_HAL_Driver/Src/stm32f4xx_hal_dma2d.c')
+    SOURCE_TREZORHAL.append('embed/trezorhal/dma2d.c')
+    CPPDEFINES_MOD += [
+        'USE_DMA2D',
+    ]
+
+# fonts
+def add_font(font_name, font, defines, sources):
```

Comment:
you can add this function to `site_scons`, see https://scons.org/doc/1.0.0/HTML/scons-user/x3577.html

Reply:
done https://github.com/trezor/trezor-firmware/commit/aea19c11bf847e8abd15133f8c84d557971e4a64